### PR TITLE
feature/driver-update

### PR DESCRIPTION
**Pull request by miloti-aliu**

### CODE REVIEW

**Comment by ardit-ubt** [comment]
@miloti-aliu your PR looks fine, but can you please just remove all the files from the 'target' folder, as usually none of those files are committed in any PR.

File: src/test/java/definitions/Hooks.java [hunk 1]
```diff
@@ -12,6 +12,9 @@
 
     @Before
     public void setup() {
+
+        System.setProperty("webdriver.gecko.driver", "C:\\repos\\serenity-bdd\\drivers\\geckodriver.exe");
```
Comment (ardit-ubt) [inline src/test/java/definitions/Hooks.java:16]: Please use relative path if possible.

**Comment by ardit-ubt** [comment]
@miloti-aliu please name your branches in present time, in your case 'updated' should go for 'update'.

**Comment by ardit-ubt** [comment]
@miloti-aliu Please decline [this PR](https://github.com/ardit-ubt/serenity-bdd/pull/10), then delete your local and remote branch 'feature/driver-update' and then rename this branch. After that we'll be able to merge changes.

**Comment by miloti-aliu** [comment]
@ardit-ubt PR number #4 reopened.

**Pull request merged**